integration/onedc_test.go

```diff
@@ -13,26 +13,26 @@ func TestOneDockerCompose(t *testing.T) {
 	f := newDCFixture(t, "onedc")
 	defer f.TearDown()
 
-	f.dockerKillAll("onedc_web")
+	f.dockerKillAll("tilt")
 	f.TiltWatch("web")
 
 	ctx, cancel := context.WithTimeout(f.ctx, time.Minute)
 	defer cancel()
 
-	f.WaitUntil(ctx, "onedc_web up", func() (string, error) {
+	f.WaitUntil(ctx, "onedc up", func() (string, error) {
```

Comment:
this gets used a few times -- worth making a helper function for?

Reply:
good idea, fixed!